src/main/java/school/lemon/changerequest/java/generics/pair/TestingApp.java

```diff
@@ -0,0 +1,32 @@
+package school.lemon.changerequest.java.generics.pair;
+
+import java.lang.reflect.Array;
+import java.util.Arrays;
+
+/**
+ * Created by Yaroslav Pavlinskiy on 22.12.2016.
+ */
+public class TestingApp {
+    public static void main(String[] args) {
+        Pair p1 = new Pair<>(1,"k1");
```

Comment:
Your pairs should be initialized with some Generics, you should not use raw notation, so please, use smth like:
```java
Pair<Integer,String> p1 = new Pair<>(1,"k1");
```